JS/Lesson26/saffet/createPostPage.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Create New Post</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+    <div class="container" id="container">
+        <h1>Posts</h1>
+
+        <form id="postForm" class="form">
+            <label for="title">Title:</label>
+            <input class="titleText" type="text" id="title" name="title">
+            <br><br>
```

Comment:
Try to avoid using `<br>`elements, they are leftovers from the past that are hard to maintain. Better add spacing using margin in CSS.